po/glossary/pt_BR.po

```diff
@@ -84,7 +84,7 @@ msgstr "Receita"
 
 #. "A debt, a financial obligation, but see also 'account type: Passive'"
 msgid "account type: Liability"
-msgstr "Passivo"
+msgstr "tipo de conta: Responsabilidade"
```

Comment:
@wtuemura Please remove `tipo de conta: ` here and later. See the [instructions in the wiki](https://wiki.gnucash.org/wiki/Translation#The_glossary_file)